independent-publisher-connectors/Forem/apiProperties.json

```diff
@@ -0,0 +1,44 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "hostName": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify the your hostname without 'https://' and no '/api/'. In the case of dev.to use 'dev.to'.",
+          "displayName": "Hostname (without 'https://' and '/api/')",
+          "tooltip": "Specify the your hostname without 'https://' and no '/api/'. In the case of dev.to use 'dev.to'."
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
```

Comment:
@Laskewitz for all our independent publishers we ask to set the "iconBrandColor" to "#da3b01". Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).